svc.go

```diff
@@ -123,7 +124,12 @@ func (as *autoNATService) handleDial(p peer.ID, obsaddr ma.Multiaddr, mpi *pb.Me
 		}
 
 		if as.config.dialPolicy.skipDial(addr) {
```

Comment:
We may skip an address because it's a relay address (e.g.). This logic will continue to dial said addresses.